stdlib/LinearAlgebra/src/bunchkaufman.jl

```diff
@@ -348,11 +348,6 @@ function ldiv!(B::BunchKaufman{T}, R::StridedVecOrMat{T}) where T<:BlasComplex
         end
     end
 end
-# There is no fallback solver for Bunch-Kaufman so we'll have to promote to same element type
-function ldiv!(B::BunchKaufman{T}, R::StridedVecOrMat{S}) where {T,S}
-    TS = promote_type(T,S)
-    return ldiv!(convert(BunchKaufman{TS}, B), convert(AbstractArray{TS}, R))
-end
```

Comment:
This is an untested method that doesn't even follow the `ldiv!` contract: when the types `TS` and `S` are different, then `R` gets copied. Usually, we don't promote `eltypes` in in-place functions.